build/tools/appveyor-test.bat

```diff
@@ -1,3 +1,12 @@
+echo.
+echo --- Running httpbin...
+echo.
+
+pip install httpbin
+start /b python -m httpbin.core
+set WX_TEST_WEBREQUEST_URL="http://127.0.0.1:5000"
```

Comment:
```suggestion
set WX_TEST_WEBREQUEST_URL=http://127.0.0.1:5000
```
Try this.

Any way to abort the appveyor build early instead of hogging it for 7+ hours?

Reply:
I can't believe I didn't notice this :-( Thanks!

> Any way to abort the appveyor build early instead of hogging it for 7+ hours?

I can cancel the builds, but I have no idea why/how is it configured because while I can do it for wx, I can't do it for other GitHub projects. And I don't see any possibility to add someone else to the project in their web UI, so I can't give you the permissions to do it neither, sorry.

Reply:
I did not mean to abort it manually. I was wondering about the previous AppVeyor build (https://ci.appveyor.com/project/wxWidgets/wxwidgets/builds/37423322)
The test script ends (it shows `!!! Non-GUI test failed`), but then the build does not finish. Maybe you know what is going on?

Maybe you need to stop `python -m httpbin.core`? We'll see what happens to this build.

Reply:
Ah, sorry for misunderstanding.

You are probably right and it times out waiting for `python` process to exit. Unfortunately I don't know how to get the PID of the process using `cmd.exe` and using `taskkill /im python` might be too drastic. But short of rewriting the build script in PowerShell (which might not be bad idea...) I don't see anything better, do you?

Reply:
You could use `before_test` and `after_test` in `AppVeyor.yml`. Install and start `httpbin`, save the pid as env var, and then stop it after testing. Then you can use PS there, and keep the bat file (for now).